recipe/build.sh

```diff
@@ -24,6 +24,20 @@ case $target_platform in
         export LIBPTHREAD=
         ./configure --disable-shared --enable-static --prefix=$PREFIX --enable-cblas --enable-threading=pthreads --enable-arg-max-hack x86_64
         make -j${CPU_COUNT}
+        echo "Contents of frame/3/gemm/bli_gemm_ker_var2.c:"
+        cat frame/3/gemm/bli_gemm_ker_var2.c
+        echo "Contents of kernels/skx/3/bli_dgemm_skx_asm_16x14.c:"
+        cat kernels/skx/3/bli_dgemm_skx_asm_16x14.c
+        echo "Compiling assimbly of frame/3/gemm/bli_gemm_ker_var2.c..."
+        echo "clang-cl -I$PREFIX\Library\include\blis /FAcs -o bli_gemm_ker_var2.s frame/3/gemm/bli_gemm_ker_var2.c"
```

Comment:
```suggestion
        echo "clang-cl -I$PREFIX\include\blis /FAcs -o bli_gemm_ker_var2.s frame/3/gemm/bli_gemm_ker_var2.c"
```

Reply:
Are you certain? We're in the windows case here (though I agree it's confusing because we're coming through `run_autotools_clang_conda_build.bat`); in the test section we're using `%LIBRARY_INC%`, which does contain `\Library`...

Reply:
Yes

Reply:
OK, not surprising that your're right, but I'm still surprised `%PREFIX%` contains `Library`. Or are we picking up `$PREFIX` instead of `%PREFIX%` - running bash on windows is confusing... 😅